frontend/src/hooks/useToken.js

```diff
@@ -9,12 +9,18 @@ export default function useToken() {
   const [token, setToken] = useState(getToken());
 
   function saveToken(token) {
+    console.log("savetoken")
+    if (token === null) {
```

Comment:
Ez biztos hogy minden esetben null? mert pl ha "" string jön vissza akkor is inkább törölni kéne a sessionStorage-et

Reply:
Jogos :smiley:  javítottam